### PR TITLE
Fix: Hide support threshold when it is negative

### LINKED ISSUE #2059: Governance. Threshold. Negative value for threshold
## Description

Negative value is shown for refereneda threshold
Screenshot 2024-08-09 at 11 45 48

## Steps to Reproduce

1. Create Origin - Auction Admin referenda on testnet
Screenshot 2024-08-09 at 11 48 36

2. Vote for it
3. Waiting until apply

## Expected Behavior
Shouldn't be negative threshold

## Actual Behavior
0.1K of -93264.26407 WND

## Environment
https://github.com/novasamatech/nova-spektr/commit/769e2c447f837eea9cdc8d946a1bd416cacb1fc2

### PR DESCRIPTION
**Pull request by Tbaut**
closes #2059

### CODE REVIEW

File: src/renderer/entities/governance/lib/opengovThresholdService.ts [hunk 1]
```diff
@@ -66,7 +66,11 @@ function getThreshold(curve: VotingCurve, input: BlockHeight, decisionPeriod: BN
     //   .checked_rounding_div(FixedI64::from(x) + *x_offset, Low)
     //   .map(|yp| (yp + *y_offset).into_clamped_perthing())
     //   .unwrap_or_else(Perbill::one)
-    return bnMin(BN_BILLION, reciprocal.factor.mul(BN_BILLION).div(div).add(reciprocal.yOffset));
+    const res = bnMin(BN_BILLION, reciprocal.factor.mul(BN_BILLION).div(div).add(reciprocal.yOffset));
+    console.log('curve', curve);
+    console.log('res', res.toString());
```
Comment (Asmadek) [inline src/renderer/entities/governance/lib/opengovThresholdService.ts]: remove unnecessary console .log

File: src/renderer/entities/governance/model/supportThreshold.ts [hunk 2]
```diff
@@ -52,6 +52,8 @@ const requestSupportThresholdsFx = createEffect(
         blockDifference: referendum.deciding?.since ? blockNumber - referendum.deciding.since : 0,
         decisionPeriod: new BN(track.decisionPeriod),
       });
+
+      console.log('result[referendum.referendumId].value', result[referendum.referendumId].value.toString());
```
Comment (Asmadek) [inline src/renderer/entities/governance/model/supportThreshold.ts]: same comments for logs

**Review by Asmadek — APPROVED** [review]

File: src/renderer/entities/governance/lib/opengovThresholdService.ts (hunk 1 above)
Comment (Tbaut) [reply src/renderer/entities/governance/lib/opengovThresholdService.ts]: oh sorry, I had forgotten to push the changes. it's done

File: src/renderer/entities/governance/model/supportThreshold.ts (hunk 2 above)
Comment (Tbaut) [reply src/renderer/entities/governance/model/supportThreshold.ts]: oh sorry, I had forgotten to push the changes. it's done

**Review by pgolovkin — APPROVED** [review]

**Pull request merged**